src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -52,10 +56,13 @@ def __init__(self, config, gpgkeyring, gpgkeyverifier, outbox_path):
         self.gpgkeyverifier = gpgkeyverifier
         self.gpgmailbuilder = gpgmailbuilder.GpgMailBuilder(
             self.gpgkeyring, self.config['main_loop_duration'])
-        self.mailsender = mailsender.MailSender(self.config)
 
         self.outbox_path = outbox_path
 
+        self.netcheck_broadcast = broadcastconsumer.BroadcastConsumer(
+            'netcheck', BROADCAST_NETCHECK_GATEWAY_CHANGED_NAME,
```

Comment:
You should camel case 'netcheck'.

Reply:
;-)

Reply:
![b5cd77c10ff34725](https://user-images.githubusercontent.com/5260293/58847055-2a2e7b00-8647-11e9-95bd-cb3b5f7ca864.jpeg)